src_py/__init__.py

```diff
@@ -28,14 +28,14 @@
 
 # Choose Windows display driver
 if os.name == "nt":
-    pygame_dir = os.path.split(__file__)[0]
+    _pygame_dir = os.path.split(__file__)[0]
```

Comment:
Another solution might be to just `del` this variable once we are done using it. But this works too